src/audio/crossover/tune/crossover_paths.m

```diff
@@ -0,0 +1,23 @@
+function crossover_paths(enable)
+
+% crossover_paths(enable)
+% enable - set to true to enable needed search path
+%          set to false to disable the search paths
+%
+
+% SPDX-License-Identifier: BSD-3-Clause
+%
+% Copyright (c) 2024, Intel Corporation. All rights reserved.
+
+	common = '../../../../tools/tune/common';
+	eq = '../../../../tools/tune/eq';
+	# After #9187 merge use this:
```

Comment:
ok, this is ok in short term, long term we will need to set a ```${prefix}``` as a base directory via cmake (and this will help with make install too). We should also obey any rules that Matlab/Octave use for libraries of functions when it comes to storage.